core/imageroot/usr/local/agent/pypkg/agent/__init__.py

```diff
@@ -456,6 +456,9 @@ def match_filter(record, clauses):
             if not 'node' in rvalue:
                 rvalue['node'] = rdb.hget(f'module/{module_id}/environment', 'NODE_ID')
 
+            if rkey.split('/')[2] != 'srv':
+                rvalue['qualifier'] = rkey.split('/')[2]
+
```

Comment:
The qualifier is [already endorsed by the docs](https://nethserver.github.io/ns8-core/modules/service_providers/#srv-keys), due to the possibility to have multiple `srv` with the same name.
Returning the information if set only adds a way to filter the information instead of nesting multiple values inside the same hash